guides/source/autoloading_and_reloading_constants.md

```diff
@@ -1313,3 +1313,68 @@ class C < BasicObject
   end
 end
 ```
+
+### Missing Singleton Methods
+
+On-demand autoloading and reopening sometimes do not work well together. This kind of
+pitfall typically happens when defining singleton methods on a module for namespace,
+Let's consider:
+
+```ruby
+# app/helpers/blog/posts_helper.rb
+module Blog
+  module PostsHelper
+  end
+end
+```
+
+```ruby
+# app/models/blog.rb
+module Blog
+  def self.table_name_prefix
+    "blog_"
+  end
+end
+```
+
+```ruby
+# app/models/blog/post.rb
+class Blog::Post < ApplicationRecord
+end
+```
+
+In the above example, you certainly expect that the table name of `Blog::Post`
+should be `blog_posts` because a singleton method called `table_name_prefix` is
+defined in `app/models/blog.rb`.
+
+However, you possibly come across an error or weird behavior which indicate that
+Rails tries to read `posts` table.
```

Comment:
indicate -> indicates

This sentence is very vague. Can you describe the behavior or a specific error that a programmer might see?